server/storage.go

```diff
@@ -561,3 +564,113 @@ func saveGDriveToken(path string, token *oauth2.Token, logger *log.Logger) {
 
 	json.NewEncoder(f).Encode(token)
 }
+
+var uplinkFailure = errs.Class("uplink failure")
```

Comment:
This could have been var `var Error = errs.Class("storj error")` if you wanted to indicate the error comes from Storj.

Reply:
adressed on upstream directly